pkg/api/aim/query/query.go

```diff
@@ -364,13 +342,36 @@ func (pq *parsedQuery) parseCompare(node *ast.Compare) (any, error) {
 		default:
 			switch right := right.(type) {
 			case clause.Column:
-				o, l, r, err := reverseComparison(op, left, right)
-				if err != nil {
-					return nil, err
-				}
-				exprs[i], err = newSqlComparison(o, l, r)
-				if err != nil {
-					return nil, err
+				switch op {
+				case ast.In:
+					return clause.Like{
```

Comment:
we should probably check that `left` is a string in those 2 cases

Reply:
done.